README.md

```diff
@@ -416,21 +416,19 @@ brew install dfed/safedi/safeditool
 
 #### Other configurations
 
-If your first-party code comprises multiple modules in Xcode, or a mix of Xcode Projects and Swift Packages, or some other configuration not listed above, you will need to utilize the `SafeDITool` command-line executable directly.
+If your first-party code comprises multiple modules in Xcode, or a mix of Xcode Projects and Swift Packages, or some other configuration not listed above, you will need to utilize the `SafeDITool` command-line executable directly as a pre-build script.
 
-The `SafeDITool` utility is designed to able to be integrated into projects of any size or shape.
-
-You can build the SafeDI tool locally by running the following command at the root of this repository:
-```zsh
-swift build -c release
+```sh
+xcrun swift run --package-path $BUILD_ROOT/../../SourcePackages/checkouts/SafeDI --scratch-path $BUILD_DIR/SafeDITool-Release -c release SafeDITool --include RELATIVE_PATH_TO_SOURCE_FILES RELATIVE_PATH_TO_MORE_SOURCE_FILES --dependency-tree-output $PROJECT_DIR/RELATIVE_PATH_TO_WRITE_OUTPUT_FILE
```

Comment:
This is a _beast_ of a command, and one that relies on implementation details of Xcode's DerivedData... however, I struggled to come up with a better way to reasonably keep the SafeDI tool version in sync with the repository's dependency. Open to suggestions here.

Folk can of course use the brew-installed `safeditool`, but that approach has the same version sync problem if you aren't on latest.

Reply:
It's a bit more nuts now, but way, way, way, way faster. e091aa033dc10db2422f6aff9e9708d50f280d73